src/components/Board.tsx

```diff
@@ -0,0 +1,116 @@
+import React, { useState, useEffect } from 'react'
+import Square from './Square'
+import lines from './winning-conditions'
+import Confetti from 'react-dom-confetti'
+import confettiConfig from './confetti-config'
+
+//Board props
+type BoardType = {
+  squares: string[][]
+  updateHistory: (squares: string[]) => void
```

Comment:
```suggestion
  updateHistory: (squares: GameState) => void
```

if you used the previous type, it would look like this